setup.py

```diff
@@ -263,6 +263,9 @@ def use_bundled_lua(path, macros):
 
     # Parse .o files from 'makefile'
     makefile = os.path.join(path, "makefile")
+    if libname == 'lua51':
+        # Lua 5.1.5 source code is downloaded from lua.org/ftp
+        makefile = os.path.join(path, "src", "Makefile")
```

Comment:
```suggestion
    # Find Makefile in subrepos and downloaded sources.
    for makefile_path in [os.path.join("src", "makefile"), os.path.join("src", "Makefile"), "makefile", "Makefile"]:
        makefile = os.path.join(path, makefile_path)
        if os.path.exists(makefile):
            break
    else:
        raise RuntimeError("Makefile not found in " + path)
```

Reply:
That would fail on Windows. Because Windows path is case insensitive, and lua_5.1.5.tar.gz also has a `Makefile` in the root directory but that file doesn't have any `.o` target.